client/src/components/v3/status/CameraStatusCol.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { Col, Table } from 'react-bootstrap';
+import OnlineStatusPill, { isOnlineT } from 'components/common/OnlineStatusPill';
+
+export interface CameraStatusProps {
+    cameraName: string,
+    isOnline: isOnlineT,
+    batteryVoltage?: number,
+    ip?: string,
+    videoFeedStatus?: string,
+};
+
+
+export default function CameraStatusCol({
+    isOnline,
+    batteryVoltage,
+    ip,
+    videoFeedStatus,
+    cameraName }: CameraStatusProps) {
+    return (
+        <Col md xl="12" className="my-2">
+            <span>
+                <b>{cameraName}</b> <OnlineStatusPill isOnline={isOnline} />
+            </span>
+
+            {/* Only show more information if the camera is online */}
+            {isOnline && <>
+                {/* IP address */}
+                <p style={{ fontSize: "0.75rem", color: "gray" }}>{ip}</p>
```

Comment:
This can be made its own component and reused in `WirelessModuleStatusCol`

Reply:
I think in this case it's quite small and I don't want to increase the size of this PR (the boilerplate with stories would be more lines of code than just this). Happy to make it a component in the future if it's used more frequently.